src/collections/_documentation/workflow/notifications/index.md

```diff
@@ -69,7 +69,7 @@ Legacy Integrations (via "Service" as any individual integration) that can send
 - Campfire*
 - IRC
 - OpsGenie
-- PagerDuty
+- [PagerDuty]({%- link _documentation/workflow/integrations/global-integrations.md -%}#pagerduty)
```

Comment:
@MimiDumpling Are there any other places that we should be linking to? I think the other sections I checked were specifically referencing the plugin so I didn't want to link from there. (unless we want to add a note about hey look this is now a global integration)

Reply:
@MeredithAnya Can we link here: https://docs.sentry.io/workflow/integrations/